package.json

```diff
@@ -7,7 +7,7 @@
     "build": "gatsby build --prefix-paths",
     "build:preview": "gatsby build",
     "now-build": "yarn build",
-    "lint": "eslint ."
+    "lint": "eslint . & markdownlint-cli2 \"**/*.{md,mdx}\" \"!.github\" \"!node_modules\""
```

Comment:
Not a blocker, but I like when the `ci.yml` workflow is split up. currently it says linting "all" but it would be more helpful if we had jobs for "eslint" and "markdownlint" so the error is more clear.